DESCRIPTION

```diff
@@ -83,7 +82,7 @@ VignetteBuilder:
     knitr
 RdMacros:
     Rdpack
-Remotes: 
+Remotes:
     github::insightsengineering/sasr
```

Comment:
unrelated, but I think we can remove this now, given that `sasr` is on CRAN

Reply:
https://github.com/openpharma/mmrm/issues/299

too small to be raised haha